stores/healthcareProfessionalsStore.ts

```diff
@@ -39,6 +39,26 @@ async function queryHealthcareProfessionals() {
     }
 }
 
+export async function getHealthcareProfessionalById(id: string) {
+    try {
+        const healthcareProfessionalId = {
```

Comment:
nit: would rename this to something like data, payload, params, etc.
since other stores are using `data` I'd recommend this:
```suggestion
        const queryData = {
```

Reply:
Is this one updated? I don't see any changes

Reply:
Yes line 44, when you click on the fixup commit you can also see the change